jangjuseong/.env.sample

```
@@ -0,0 +1,11 @@
+DATABASE_URL = "rbdms://username:password@host:port/database-name"
```

주성님! 오탈자가 있는 것 같습니다!

RDBMS란 Relational DataBase Management System의 약자로 협업하는 다른 팀 동료에게 혼선을 주지 않도록 수정해주면 좋을 것 같습니다.

```suggestion
DATABASE_URL = "rdbms://username:password@host:port/database-name"
```